stages/org.osbuild.dracut

```diff
@@ -201,15 +203,35 @@ def main(tree, options):
 
     opts += extra
 
-    for kver in kernels:
-        print(f"Building initramfs for {kver}", file=sys.stderr)
-
-        subprocess.run(["/usr/sbin/chroot", tree,
-                        "/usr/bin/dracut",
-                        "--no-hostonly",
-                        "--kver", kver]
-                       + opts,
-                       check=True)
+    mounts = [
+        {"path": "/dev", "bind": True, "ro": False, "mode": 0o755},
+        {"path": "/proc", "bind": True, "ro": True, "mode": 0o555},
+        {"path": "/sys", "bind": True, "ro": True, "mode": 0o555},
+    ]
+    with MountGuard() as mounter:
+        for mount in mounts:
+            ro = mount["ro"]
```

Comment:
Given that this pattern is used three time, I wonder if it could become a helper in `osbuild.util.mnt` - maybe something like `mnt.bind_mount_for_chroot(MountGuard, tree)` (feel free to pick a better name, naming is hard :)

Reply:
I like the name you suggested, it's very clear what the function is doing :)

Reply:
Given that it's now only a single place I guess it's fine and I will close this comment :)